pkg/driver/controller.go

```diff
@@ -635,3 +747,38 @@ func (cs *controller) validateVolumeCreateReq(req *csi.CreateVolumeRequest) erro
 	}
 	return nil
 }
+
+func (cs *controller) validateSnapshotRequest(req *csi.CreateSnapshotRequest) error {
```

Comment:
I don't see thais getting called in the CreateSnapshot request. Where we are validating snapshot request?

Reply:
Updated.